### PR TITLE
Containerize Dark Blue job (DBAI-22)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR aims to resolve [DBAI-22](https://mlit.atlassian.net/jira/software/projects/DBAI/boards/89?selectedIssue=DBAI-22).
To Do
- [x] Add `Dockerfile` with`development` and `production` stages
- [x] Support `ssh`/`sftp` (at least for `development`)
- [x] Pick up minor versions of dependencies due to `Gemfile.lock` regeneration.  
- [x] Modify `README.md` to incorporate `docker compose` as an installation/usage approach

### CODE REVIEW

File: run_dark_blue.rb [hunk 1]
```diff
@@ -7,72 +7,65 @@
...
+        base_url: api_config.base_url,
+        api_key: api_config.api_key,
+        username: api_config.username
+      )
+      remote_config = arch_config.remote
+      remote_client = RemoteClient::RemoteClientFactory.from_config(
+        type: remote_config.type,
+        settings: remote_config.settings
+      )
+      digital_objects += Archivematica::ArchivematicaService.new(
```
Comment (respinos) [inline run_dark_blue.rb:44]: You're collecting all the possible bags before uploading anything?

Comment (respinos) [reply run_dark_blue.rb:44]: ¯\_(ツ)_/¯ let's see how this works! Our previous APTrust uploading definitely had to process each collection at a time.

File: lib/remote_client.rb [hunk 2]
```diff
@@ -8,6 +8,18 @@ module RemoteClient
...
+      command_str = [array_of_commands].join(" ")
+      `bash  -c \"#{command_str}\"`
+      p command_str
+      p $?
+      if !$?.success?
+        raise RemoteClientError, "Error occurred during SFTP process: #{$?.exitstatus}"
+      end
+    end
+  end
+
```
Comment (ssciolla) [inline lib/remote_client.rb:22]: Ack, I need to remove this.

**Review by conorom — APPROVED** [review]

**Comment by ssciolla** [comment]
Thanks, @conorom!

**Pull request merged**